web/src/components/VendorHeader.js

```diff
@@ -28,6 +28,9 @@ const VendorHeader = ({setLogoutState}) => {
             <Link to='/vendorprofile' className='vendor-header-category px-4 py-3 mx-2 rounded'>
                 Profile
             </Link>
+            <Link to='/addproduct' className='vendor-header-category px-4 py-3 mx-3 rounded'>
```

Comment:
Very minor css issue. ><
The vendor header links here had their styles updated recently (their horizontal margin slightly lowered, so that more links can comfortably fit on the screen). In other words, other links have mx-2 while this one has mx-3.

Reply:
Updated it